api/src/main/java/io/grpc/ClientCall.java

```diff
@@ -141,6 +141,11 @@ public void onMessage(T message) {}
      * An additional block of trailer metadata may be received at the end of the call from the
      * server. An empty {@link Metadata} object is passed if no trailers are received.
      *
+     * <p>If uncaught exception is thrown in `onClose`, gRPC will fail to notify the application
```

Comment:
I think this needs to be in ForwardingClientCall or friend. This is not part of `ClientCall` itself.

Reply:
Sorry, I just realized this is duplicated work as https://github.com/grpc/grpc-java/pull/6107. It seems that merge request is block  for a long time, is there any plan to move forward with this? Do you want me to take #6107 over?

Reply:
Ah, yes, that is similar. @dapengzhang0, what do you think?

Reply:
I don't mind if @wanyingd1996 takes  #6107 over. Actually forgot about that PR

Reply:
I think the text I suggested in dapengzhang0's PR is still appropriate. When talking just about interceptors (as was the case with language like "gRPC will fail to notify the application"), then this isn't the correct spot for it.